src/pages/cryptoPage.jsx

```diff
@@ -0,0 +1,56 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { getCryptoPrices, coinInfo } from '../redux/crypto/cryptoSlice';
+import CryptoInfo from './cryptoInfo';
+import CryptoImage from '../assets/CRYPTOunsplash.webp';
+
+const CryptoPage = () => {
+  const dispatch = useDispatch();
+  const cryptoArr = useSelector((state) => state.crypto.cryptoArr);
+  const loading = useSelector((state) => state.crypto.loading);
+
+  useEffect(() => {
+    if (cryptoArr.length === 0) {
+      dispatch(getCryptoPrices());
+    }
+  }, [dispatch, cryptoArr]);
+  const coinInfoHandel = (id) => {
+    dispatch(coinInfo(id));
+  };
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+  if (cryptoArr.length > 1) {
+    return (
+      <>
+        <h2 className="crypto-h2">Crypto Prices in USD</h2>
+        <section>
+          <img src={CryptoImage} alt="" />
+        </section>
+        <div>
+          {cryptoArr ? (
+            <div className="crypto-container">
+              {cryptoArr.map((crypto) => (
+                <button type="button" onClick={() => coinInfoHandel(crypto.id)} key={crypto.id} className="single-crypto" id={crypto.id}>
+                  <img src={crypto.image} alt="coinImage" />
+                  <p key={crypto.id}>
+                    {crypto.name}
+                    <br />
+                    Price: $
+                    {crypto.current_price}
+                  </p>
+                </button>
+              ))}
+            </div>
+          ) : (
+            <div>Loading data...</div>
+          )}
+        </div>
+
+      </>
+    );
+  } return (<CryptoInfo />);
+};
```

Comment:
- [ ] You did really a great job on this project but you missed a major requirement. Please add a feature on the home page to filter cryptos. (You should include a search feature on the page).
![image](https://user-images.githubusercontent.com/83803414/193900059-ae0b963d-7671-4e1c-910c-bc3ad366aaa2.png)

Reply:
Hi @thecodechaser thanks for the review 
I believe I added the filter function in the slice as the following
`  reducers: {
    coinInfo: (state, action) => {
      state.cryptoArr = state.cryptoArr.filter((x) => x.id === action.payload);
    },
`
so it will filter the input and only show one required element